src/openrct2-ui/windows/TrackDesignManage.cpp

```diff
@@ -233,7 +233,7 @@ namespace OpenRCT2::Ui::Windows
         auto ft = Formatter();
         ft.Add<const utf8*>(_trackDesignFileReference->name.c_str());
         DrawTextWrapped(
-            rt, windowPos + ScreenCoordsXY{ (kWindowSizeDeletePrompt.width / 2), ((kWindowSizeDeletePrompt.height / 2) - 9) },
+            rt, windowPos + ScreenCoordsXY{ (kWindowSizeDeletePrompt.width / 2), ((kWindowSizeDeletePrompt.height / 2) - 3) },
```

Comment:
Changing the offset from 9 to 3 is not going to have the desired effect in all situations. The Y coordinate should be expressed using the `.bottom` of the title widget instead.